internal/dag/status_test.go

```diff
@@ -4480,12 +4480,19 @@ func TestGatewayAPIHTTPRouteDAGStatus(t *testing.T) {
 			RouteParentStatuses: []*gatewayapi_v1beta1.RouteParentStatus{
 				{
 					ParentRef: gatewayapi.GatewayParentRef("projectcontour", "contour"),
-					Conditions: []metav1.Condition{{
-						Type:    string(gatewayapi_v1beta1.RouteConditionAccepted),
-						Status:  contour_api_v1.ConditionTrue,
-						Reason:  string(gatewayapi_v1beta1.RouteReasonAccepted),
-						Message: "Accepted HTTPRoute",
-					}},
+					Conditions: []metav1.Condition{
+						{
+							Type:    string(gatewayapi_v1beta1.RouteConditionResolvedRefs),
+							Status:  contour_api_v1.ConditionTrue,
+							Reason:  string(gatewayapi_v1beta1.RouteReasonResolvedRefs),
+							Message: "References Resolved",
+						},
```

Comment:
Since you introduced a helper for defining this condition (💯), let's use it in all of these test cases instead of copy-pasting the struct literal.

Separately (not in this PR), we should circle back and do the same for the Accepted condition

Reply:
> Separately (not in this PR), we should circle back and do the same for the Accepted condition

done,  i will send a new pr to do the `Accepted` condition later.